scripts/generateContributors.js

```diff
@@ -0,0 +1,19 @@
+import { Octokit } from "octokit";
+import fs from "node:fs";
+import path from "node:path";
+import url from "node:url";
+
+async function generateContributors() {
+  const octokit = new Octokit({ auth: process.env.GITHUB_ACCESS_TOKEN });
+
+  const results = await octokit.paginate(octokit.rest.repos.listContributors, {
+    owner: "grain-lang",
+    repo: "grain",
```

Comment:
Ah true good point, should I just sum together the count of contributions across all repos in the grain-lang organization?

Reply:
I think that would be reasonable I didn't see a good way todo that in octokit though (I might have just missed it), I think if we need todo it manually it would be fine to just count the major repos. (The website, binaryen.ml, the main repo, maybe awesome-grain).